freqtrade/worker.py

```diff
@@ -26,12 +27,15 @@ def __init__(self, args: Dict[str, Any], config: Dict[str, Any] = None) -> None:
         """
         Init all variables and objects the bot needs to work
         """
-        logger.info('Starting worker %s', __version__)
+        logger.info(f"Starting worker {__version__}")
 
         self._args = args
         self._config = config
         self._init(False)
 
+        self.last_throttle_start_time: float = 0
+        self._heartbeat_msg: float = 0
```

Comment:
i know you only moved this here - but looking at the variable name now i did not really expect a time as content.

Maybe we can use `self._last_heartbeat_ts` instead ?

Reply:
Already done in the upcoming commits (not in this PR)